src/niswitch/metadata/enums.py

```diff
@@ -987,5 +987,21 @@
                 'value': 2
             }
         ]
+    },
+    'WireMode': {
```

Comment:
Another new enum. I found the internal discussion of it. I think this one's okay.

Reply:
Reopening conversation, in case we decide we can/should omit it along with other new grpc enums.

Reply:
I think we should set 'codgen_method': 'no' for this, but it's not being added to the generated code, so it probably doesn't need to hold up this PR.

Reply:
If it's internal only, I'm ok to merge.

Reply:
I have a [PR](https://ni.visualstudio.com/DevCentral/_git/ni-central/pullrequest/373719) ready to commit that will allow this. After that, I'll go apply it to all of those enums. But I'll make those changes separate from this PR.